src/main/kotlin/event/player/PlayerInteract.kt

```diff
@@ -20,4 +22,11 @@ class PlayerInteract : Listener {
             }
         }
     }
+
+    @EventHandler
+    fun onPlayerInteract(event: PlayerInteractEntityEvent) {
+        if (event.hand.equals(org.bukkit.inventory.EquipmentSlot.HAND)) { // The reason for this is really funny
+            itemframeInteractEvent(event)
+        }
+    }
```

Comment:
I'd prolly move this to its own PlayerInteractEntity.kt class to keep it to one listener per file.